fastlane/Fastfile

```diff
@@ -66,68 +86,11 @@ platform :ios do
     )
   end
 
-  desc "Replace version number in project and supporting files"
-  lane :replace_version_number do |options|
-    new_version_number = options[:version]
-    UI.user_error!("missing version") unless new_version_number
-    previous_version_number = current_version_number
-    
-    version_number_without_prerelease_modifiers = new_version_number.split("-")[0]
-    increment_version_number(version_number: version_number_without_prerelease_modifiers)
```

Comment:
I hadn't noticed this was an iOS only operation. This increments the version in the xcodeproj in a smarter way, using some specialized tools. 

I substituted this with our more "manual" version replacement (changing all occurences of the old version number with the new one). I tested it a few times, and it seems to generate the same changes, but lmk if you don't like that and we can keep this task as sdk-dependant

Reply:
A more platform-agnostic approach like you describe makes sense to me. In the future it might even make sure to extract out `ENV["FILES_TO_UPDATE_VERSION_STRING"]` into separate file that gets loaded.